jest.config.ci.js

```diff
@@ -15,9 +15,6 @@ module.exports = {
       'jest-junit',
       {outputDirectory: 'reports/junit', outputName: 'js-test-results.xml'},
     ],
-    [
-      'jest-silent-reporter',
-      {showPaths: true, showWarnings: true, useDots: true},
-    ],
+    'default',
```

Comment:
This should be reverted as it was just for testing - I'll drop the commit shortly, provided the rebase was successful.

Reply:
yes please 😀